packages/contentstack-utilities/src/date-time.ts

```diff
@@ -0,0 +1,15 @@
+export const formatDate = (date: Date) => {
+  return [
+    date.getFullYear(),
+    (date.getMonth() + 1).toString().padStart(2, '0'),
+    date.getDate().toString().padStart(2, '0'),
+  ].join('');
+};
+
+export const formatTime = (date: Date) => {
+  return [
+    date.getHours().toString().padStart(2, '0'),
+    date.getMinutes().toString().padStart(2, '0'),
+    date.getSeconds().toString().padStart(2, '0'),
+  ].join('');
+};
```

Comment:
**General recommendations:**
- Use string concatenation for small strings.
- Use array join for large strings.